recipes/conda-ecosystem-site-packages-isolation/meta.yaml

```
@@ -0,0 +1,28 @@
+package:
+  name: conda-ecosystem-site-packages-isolation
+  version: 0.1.0
+
+build:
+  number: 0
+
+test:
+  requires:
+    - python
+  commands:
+    - python -c "import sys, site; sys.exit(0 if site.ENABLE_USER_SITE==False else 1)"
+
+about:
+  home: http://conda-forge.org/
+  summary: Prevent the environment's Python from searching for packages outside the environment
+  description: >
+    With respect to the list of locations searched by Python for packages, mimic the
+    default behavior of virtual environments created by Python's built-in "venv" module.
+    Python in the conda environment gets isolated from system site-packages, including user
+    site-packages.
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - itcarroll
```

I'm happy to be added as a maintainer (I also maintain the related [`conda-ecosystem-user-package-isolation-feedstock`](https://github.com/conda-forge/conda-ecosystem-user-package-isolation-feedstock)).

```suggestion
    - itcarroll
    - mfansler
```